### ISSUE TITLE
Create gifs of embedding layer changes through epochs

### ISSUE THREAD

**Issue by minztmyo** [minztmyo/keras_demo#3]
@DavinDillon Whenever available, could you wrap the auto encoder into a function, and loop it around a list of parameters, and create the gifs of the changes in latent layer.

1) Choose 3 optimizers
2) Choose 3 Loss Functions
3) Choose 2 Output layer Loss Function 

Unless it's easier for you, there is no need to do a combination of the above options. Meaning 7 gifs will suffice (no need for 18 gifs).
